vendor/typesafe-config-patches/0001-Fix-tests-for-Gradle-build.patch

```diff
@@ -0,0 +1,64 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jason Penilla <11360596+jpenilla@users.noreply.github.com>
+Date: Tue, 7 Nov 2023 13:40:03 -0700
+Subject: [PATCH] Fix tests for Gradle build
+
+
+diff --git a/.gitignore b/.gitignore
+index 057f41d2039c55b862cb2051e0c14ea7e9e2c959..9210d06996180f942da21ca55daeb439d7a4c46a 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -8,3 +8,4 @@
+ target/
+ /bin
+ /node_modules
++/config/build
+diff --git a/config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala b/config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala
+index 350af6f9e9256a34dcac7f22d651299a570bfaba..6e51d2c72effff2d3294077224153e3fc1b7bca0 100644
+--- a/config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala
++++ b/config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala
+@@ -881,25 +881,25 @@ class ConfigTest extends TestUtils {
+         val o1 = conf.getValue("ints.fortyTwo").origin()
+         // the checkout directory would be in between this startsWith and endsWith
+         assertTrue("description starts with resource '" + o1.description + "'", o1.description.startsWith("test01.conf @"))
+-        assertTrue("description ends with url and line '" + o1.description + "'", o1.description.endsWith("/config/target/test-classes/test01.conf: 3"))
```

Comment:
Is it worth changing the output directories of the processResource task rather than having to change tests?

Reply:
`processTestResources` output didn't want to change for me by setting `destinatonDir`, not sure why... I don't think this patch is really that big of a deal though, if the project were forked normally and converted to Gradle I think this is the path to take, also I think it's unlikely upstream will be adding many new tests we have to change.

Reply:
sounds good to me